wp-content/themes/twentyfifteen/js/functions.js

```diff
@@ -175,4 +175,8 @@
 		}
 	} );
 
+	$( document ).ready(
+		alert("El Granto wuz here.");
```

Comment:
Just kidding. I know you were just playing. I wanted to show what it looks like when someone comments on a line of code.